recipes/bison/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import glob
+import os
+import shutil
+
+
+class ConanFileDefault(ConanFile):
+    name = "bison"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.gnu.org/software/bison/"
+    description = "Bison is a general-purpose parser generator"
+    topics = ("conan", "bison", "parser")
+    license = "GPL-3.0-or-later"
+    _source_subfolder = "source_subfolder"
+    requires = ("m4/1.4.18", "flex/2.6.4")
```

Comment:
Does Bison really require Flex to be able to run? AFAIK, `yylex` function works as an interface between lexical analyzer and syntax analyzer: https://www.gnu.org/software/bison/manual/html_node/Lexical.html , so it's completely fine to write one by hand.

Reply:
bincrafters recipe requires m4_installer and flex_installer, so they should probably be build_requires indeed, thanks !